plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/local/TestLocalFileSystemExchangeManager.java

```diff
@@ -25,6 +25,7 @@
     protected ExchangeManager createExchangeManager()
     {
         return new FileSystemExchangeManagerFactory().create(ImmutableMap.of(
-                "exchange.base-directory", System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager"));
+                "exchange.base-directory", System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager",
+                "exchange.sink-target-file-size", "10MB"));
```

Comment:
I wonder if it makes sense to set it to something lower (e.g.: `10KB` - `100KB`, depending on the average file size when testing?) to make sure the logic is validated during testing.

Reply:
https://github.com/trinodb/trino/blob/9c3961712ca813dbac303a239ac98921d5e27489/plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/AbstractTestExchangeManager.java#L168 will exercise the split code path.